docker-compose/run.sh

```diff
@@ -2,10 +2,18 @@
 
 set -eux
 
-worker="worker-ubuntu16-04"
+worker_volume="buildbarn-worker-ubuntu16-04"
+tini_volume="buildbarn-tini"
 
-sudo rm -rf bb "${worker}"
-mkdir -m 0777 "${worker}" "${worker}/build"
-mkdir -m 0700 "${worker}/cache"
+docker volume rm "${worker_volume}" || true
+docker volume rm "${tini_volume}" || true
+
+docker volume create "${worker_volume}"
+docker volume create "${tini_volume}"
+
+docker run --rm \
```

Comment:
Couldn't you put that in the compose file as well ?

Reply:
Maybe for the volume that will get tini installed into it, i.e. just have it destroyed in the script but created automatically by the compose file. (The intent of destroying the volumes is to replicate the behavior of the `sudo rm -rf` currently in this script.)

For the worker volume, it would need to be created in this script in order to be able to then do the `mkdir` calls inside the busybox container. (I wish docker-compose had init containers like Kubernetes, [but alas it does not](https://github.com/docker/compose/issues/6855).)

Reply:
Why not use the [`command`](https://docs.docker.com/compose/compose-file/#command) for that then? Wouldn't that work, and reduce the size of this script?

Reply:
Mainly style. Using `command` to add an init command means having to replicate the call to the image's entrypoint which requires prior knowledge of it. I prefer to not disturb the entrypoint configuration where possible since it is one of the details encapsulated by an image. In this case, the `run.sh` script is necessary any way to delete a subset of volumes, so leaving init code in the shell script is my preferred solution.

That said, I'm fine with conforming to the project's style. Thoughts?

Reply:
Moreover, assuming we would want to do the `mkdir` calls in the worker image, the `buildbarn/bb-worker:20200528T061211Z-4dfbe5b` image does not even include `sh` or `bash`:

```
vagrant@vagrant:/src/toolchain$ docker run -t -i --rm --entrypoint=/bin/sh buildbarn/bb-worker:20200528T061211Z-4dfbe5b
docker: Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"/bin/sh\": stat /bin/sh: no such file or directory": unknown.
```

Reply:
What I meant with using `command` would something like:
```
  worker-setup:
    image: busybox:latest
    command:
    - /bin/sh
    - -c
    - 'mkdir -p /worker/build /worker/cache'
    volumes:
    - buildbarn-worker-ubuntu16-04:/worker
```
And let the runner installer depends on that?